docs/topics/proc_viewing-the-booster-source-code-and-readme.adoc

```diff
@@ -1,5 +1,6 @@
 [id='viewing-the-booster-source-code-and-readme_{context}']
-= Viewing the booster source code and README
+[id='viewing-the-example-source-code-and-readme_{context}']
+= Viewing the example source code and README
```

Comment:
Same here. 
Viewing the example source code and README--> Viewing the example application source code and README

Reply:
Thanks. This should be better now.